Persephone/MPDClient/Models/MPDAlbum.swift

```diff
@@ -12,5 +12,13 @@ extension MPDClient {
   struct MPDAlbum: Equatable {
     let title: String
     let artist: String
+    var date: String?
+    var path: String?
+
+    static func == (lhs: MPDAlbum, rhs: MPDAlbum) -> Bool {
+      return (lhs.title == rhs.title) &&
+        (lhs.artist == rhs.artist) &&
+        (lhs.date == rhs.date)
```

Comment:
This is a preference thing, but I believe you could eliminate the surrounding parentheses here.

Reply:
Should `path` also be part of the equality check?

Reply:
It's not strictly necessary. If the title, artist and date all match then the path will too.